doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,54 @@
+"""
+===============================================
+Reconstruct dust-covered image using inpainting
+===============================================
+
+It is possible that dust gets accumulated on the reference mirror and causes
+dark spots to appear on direct images. This example reproduces the steps taken
+to perform OCT (Optical Coherence Tomography [1]_) dust removal in images.
+This application was first discussed by Jules Scholler in [2]_.
+
+.. [1] Vinay A. Shah M.D. (2015)
+       `Optical Coherence Tomography <https://eyewiki.aao.org/Optical_Coherence_Tomography#:~:text=3%20Limitations-,Overview,at%20least%2010%2D15%20microns.>`_,
+       American Academy of Ophthalmology.
+.. [2] Jules Scholler (2019) "Image denoising using inpainting":
+       `<https://www.jscholler.com/2019-02-28-remove-dots/>`_
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import imageio.v3 as iio
+import plotly.express as px
+
+
+#################################################################################
+# The dataset that we are using in this example is an image sequence showing
```

Comment:
```suggestion
# Load image
# =========
# The dataset that we are using in this example is an image sequence showing
```

Reply:
Thanks for the suggestion! Here, the reason I haven't added a sub-heading because our gallery examples (for instance, https://scikit-image.org/docs/stable/auto_examples/applications/plot_fluorescence_nuclear_envelope.html) don't usually have an introductory sub-heading. What are your thoughts?

Reply:
Although, I went through [this example](https://scikit-image.org/docs/stable/auto_examples/applications/plot_3d_structure_tensor.html#sphx-glr-auto-examples-applications-plot-3d-structure-tensor-py) that does use a sub-heading, so I will add one as well! :)